pyseir/load_data.py

```diff
@@ -399,11 +398,16 @@ def load_new_case_data_by_fips(
 
 
 def get_hospitalization_data():
+    """
+    Since we're using this data for hospitalized data only, only returning
+    values with hospitalization data.  I think as the use cases of this data source
+    expand, we may not want to drop. For context, as of 4/8 607/1821 rows contained
```

Comment:
Did you mean: as of June 11 the timeseries per region has hospitalization data for April 8? Or the latest data as of June 8?

Reply:
Oh, this isn't a new comment :-)